src/SDKs/Batch/DataPlane/Azure.Batch/Azure.Batch.csproj

```diff
@@ -10,7 +10,7 @@
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <PackageId>Azure.Batch</PackageId>
     <Description>This client library provides access to the Microsoft Azure Batch service.</Description>
-    <VersionPrefix>8.1.0</VersionPrefix>
+    <VersionPrefix>8.1.1</VersionPrefix>
```

Comment:
@matthchr add packageReleaseNotes for this release.

Reply:
@shahabhijeet We already have it, it's just in a lower `<PropertyGroup>`

It says:     `<PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-dataplane-changelog</PackageReleaseNotes>`